chatterino.pro

```diff
@@ -159,6 +159,7 @@ SOURCES += \
     src/controllers/highlights/HighlightModel.cpp \
     src/controllers/highlights/HighlightPhrase.cpp \
     src/controllers/highlights/UserHighlightModel.cpp \
+    src/controllers/ignores/IgnoreController.cpp \
```

Comment:
Did you forget to include `src/controllers/ignores/IgnoreController.hpp`? Try running `./tools/update-filelist.py`

Reply:
Row 391 has the header file - the header file was not new